### ISSUE TITLE
dt = 10 lead to phase plot fail

### ISSUE THREAD

**Issue by withr** [tgouhier/biwavelet#72]
when I using time interval equal 10, I can't plot the phase for wtc, here is an example: 

library(biwavelet)

n = 200
x = sin(seq(1, 10*pi, length.out = n)) + rnorm(n)/10
y = cos(seq(1, 10*pi, length.out = n)) + rnorm(n)/10

plot(x, type = "l", col = "green"); lines(y, col = "red")

WC.1 <- wtc(cbind(1*seq(1, n), x),  cbind(1*seq(1, n), y))

plot(WC.1, plot.phase = TRUE)

WC.10 <- wtc(cbind(10*seq(1, n), x), cbind(10*seq(1, n), y))

plot(WC.10, plot.phase = TRUE)

**Comment by vsimko** [comment]
I couldn't replicate the problem. Try using the latest version (currently v0.20.15).
Here is what I did:
```r
library(biwavelet)  ## biwavelet 0.20.15 loaded.
n <- 200
x <- sin(seq(1, 10 * pi, length.out = n)) + rnorm(n) / 10
y <- cos(seq(1, 10 * pi, length.out = n)) + rnorm(n) / 10
plot(x, type = "l", col = "green")
lines(y, col = "red")
```
![image](https://cloud.githubusercontent.com/assets/7080773/23476189/41fb95f4-feba-11e6-87e7-0bcdb83dc970.png)

```r
WC.1 <- wtc(cbind(1 * seq(1, n), x), cbind(1 * seq(1, n), y))
plot(WC.1, plot.phase = TRUE)
```
![image](https://cloud.githubusercontent.com/assets/7080773/23476224/5d321dde-feba-11e6-94b5-b2e041d2ca84.png)

```r
WC.10 <- wtc(cbind(10 * seq(1, n), x), cbind(10 * seq(1, n), y))
plot(WC.10, plot.phase = TRUE)  
```
![image](https://cloud.githubusercontent.com/assets/7080773/23476294/93353e2a-feba-11e6-8035-d354671810a2.png)

**Comment by ecorov** [comment]
Well, that's strange. I use the version 0.20.11, I think that's the latest. 

Here is the figure I got:

![rplot](https://cloud.githubusercontent.com/assets/20317526/23479146/cd8dae46-fec3-11e6-929a-6982ecdc424d.png)

One more thing: why the significance area shrinked when the dt changed from 1 to 10?

**Comment by vsimko** [comment]
try installing the latest version from github which is always ahead of the CRAN version:

```r
install.packages("devtools")
devtools::install_github("tgouhier/biwavelet")
```

**Comment by ecorov** [comment]
Yes, I got the same result as yours using version 0.20.15. Just one more question: why the significant area shrinked when change dt from 1 to 10?

**Comment by vsimko** [comment]
I would guess that it is because the numbers are 10x larger :-) but maybe @tgouhier can elaborate on this.

**Comment by withr** [comment]
I feel the "dt" shouldn't affect the result, just for plotting purpose.
